knimin/tests/test_ag_pulldown.py

```diff
@@ -73,6 +73,11 @@ def test_post(self):
                        ' for file download. It may take a while with many '
                        'barcodes.</h3>'), response.body)
 
+        data = {'external': 'ab', 'external': 'cd'}
```

Comment:
Sometimes I hate tornado but I think if you do:
```python
from json import dumps

{'external': dumps(['foo', 'bar'])}
```
should work. Basically the problem is that the parameters do not get serialized correctly in the tornado test environment.